scripts/test/generate-image.sh

```diff
@@ -136,7 +119,7 @@ Description=Regenerate grub scripts, disable itself and then shut down.
 
 [Service]
 Type=oneshot
-ExecStart=sh -c "grub-install && update-grub && systemctl disable mender-regenerate-grub-and-shutdown.service && poweroff"
+ExecStart=sh -c "grub-install && grub-install --removable && update-grub && systemctl disable mender-regenerate-grub-and-shutdown.service && poweroff"
```

Comment:
Hah, never thought of running systemctl from a service file before

Reply:
I'm sure Lennart would be proud! (or not...)